pkg/controller/controller.go

```diff
@@ -297,6 +297,12 @@ func (cont *AciController) Run(stopCh <-chan struct{}) {
 	}
 	cont.log.Info("ApicRefreshTimer conf is set to: ", refreshTimeout)
 
+        // Bailout if the refreshTimeout is more than 12Hours
```

Comment:
Can we validate this in the acc-provision? Instead of panic, we should just cap the refresh at (slightly less than) the max here.

Reply:
@jojimt Actual refresh cap is 24Hrs from APIC side, but suggested limit is 12Hrs;
Are you saying instead of panic, just set it to 12*60*60 if its more than that and perhaps log a message indicating that?

Reply:
Also, currently we don't have the way to config in acc-provision. I plan to add the support and yes, will validate on acc-provision side as well.

Reply:
> @jojimt Actual refresh cap is 24Hrs from APIC side, but suggested limit is 12Hrs;
> Are you saying instead of panic, just set it to 12_60_60 if its more than that and perhaps log a message indicating that?

Yes, exactly(just a one time msg).